services/galley/src/Galley/API/Public.hs

```diff
@@ -407,6 +433,26 @@ sitemap = do
     returns (ref Model.customBackend)
     response 200 "Custom backend" end
 
+  get "/teams/:tid/features/search-visibility" (continue Teams.getTeamSearchVisibilityEnabledH) $
+    zauthUserId
+      .&. capture "tid"
+      .&. accept "application" "json"
+  document "GET" "getTeamSearchVisibilityEnabled" $ do
+    summary "Shows whether Custom Search Visibility feature is enabled for team"
+    parameter Path "tid" bytes' $
+      description "Team ID"
+    returns (ref Model.teamSearchVisibilityEnabled)
+    response 200 "Search Visibility status" end
+  get "/custom-backend/by-domain/:domain" (continue CustomBackend.getCustomBackendByDomainH) $
```

Comment:
Not sure how this ended up here, it's not new in this PR?

Reply:
I was wondering myself, but should have wondered harder.  fixed here: e8786e7